test/unit/Order/PaymentDataTest.php

```diff
@@ -19,7 +19,7 @@ public function testToJson()
     public function testFromJson()
     {
         $payment = PaymentData::fromJson(self::getPaymentJson());
-        self::assertEquals("string", $payment->getAccountStatus());
+        self::assertEquals(null, $payment->getAccountStatus());
```

Comment:
That seems cleaner https://phpunit.de/manual/6.5/en/appendixes.assertions.html#appendixes.assertions.assertNull